Packs/CiscoEmailSecurity/pack_metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Cisco Email Security (Beta)",
```

Comment:
not sure if the name of the pack should contain beta, please double check

Reply:
I checked, and it seems it should contain beta